packages/sources/cryptoapis-v2/src/config.ts

```diff
@@ -10,3 +10,28 @@ export const makeConfig = (prefix?: string): Config => {
   config.api.baseURL = 'https://rest.cryptoapis.io'
   return config
 }
+
+export const BLOCKCHAIN_NAME_BY_TICKER = {
+  btc: 'bitcoin',
+  eth: 'ethereum',
+  ltc: 'litecoin',
+  etc: 'ethereum-classic',
+  bch: 'bitcoin-cash',
+  dash: 'dash',
+  doge: 'dogecoin',
+} as const
+export type BlockchainTickers = keyof typeof BLOCKCHAIN_NAME_BY_TICKER
+
+export function isCoinType(key: string): key is keyof typeof BLOCKCHAIN_NAME_BY_TICKER {
+  return !!BLOCKCHAIN_NAME_BY_TICKER[key.toLowerCase() as BlockchainTickers]
+}
+export const CHAIN_KEYS = ['mainnet', 'testnet'] as const
+export type ChainType = typeof CHAIN_KEYS[number]
+export function isChainType(key: string): key is ChainType {
+  return CHAIN_KEYS.includes(key as ChainType)
+}
+
+export const TESTNET_BLOCKCHAINS: { [ticker: string]: string } = {
```

Comment:
```suggestion
export const TESTNET_BLOCKCHAINS_BY_PLATFORM = {
```
Naming suggestion and type removal to keep it in line with LN#14